mkosi.conf.d/20-opensuse.conf

```diff
@@ -0,0 +1,58 @@
+[Match]
+Distribution=opensuse
+
+[Content]
+Packages=
+        attr
+        bash-completion
+        bc
+        ca-certificates
+        coreutils
+        diffutils
+        findutils
+        fio
+        gawk
+        gdb
+        git-core
+        grep
+        gzip
+        hostname
+        hwloc
+        indent
+        kbd
+        kexec-tools
+        less
+        libcap-ng-utils
+        libelf1
+        lvm2
+        make
+        man
+        mtools
+        nano
+        nftables
+        nvme-cli
+        openssh-server
+        openssl
+        pam
```

Comment:
No need to install pam explicitly it gets pulled in as a dep by everything else already